cmd/build.go

```diff
@@ -55,20 +58,51 @@ func newBuildCmd() *cobra.Command {
 	var srcURL string
 	var optionEnv map[string]string
 	var envFilePath string
+	var optionMeta string
+	var metaFilePath string
 
 	buildCmd := &cobra.Command{
 		Use:   "build [job name]",
 		Short: "Run screwdriver build.",
 		Long:  `Run screwdriver build of the specified job name.`,
-		Args:  cobra.ExactArgs(1),
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return fmt.Errorf("accepts 1 arg(s), received %d", len(args))
+			}
+			if optionMeta != "" && metaFilePath != "" {
+				return errors.New("can't pass the both options `meta` and `meta-file`, please specify only one of them")
+			}
+			return nil
+		},
 		Run: func(cmd *cobra.Command, args []string) {
+			var err error
+
 			if envFilePath != "" {
-				err := mergeEnvFromFile(&optionEnv, envFilePath)
+				err = mergeEnvFromFile(&optionEnv, envFilePath)
 				if err != nil {
 					logrus.Fatal(err)
 				}
 			}
 
+			metaJSON := []byte("{}")
+			if optionMeta != "" {
+				metaJSON = []byte(optionMeta)
+			} else if metaFilePath != "" {
+				metaJSON, err = ioutil.ReadFile(metaFilePath)
+
+				if err != nil {
+					logrus.Fatalf("failed to read meta-file %s: %s", metaFilePath, err.Error())
```

Comment:
Is it better to use "%v"?
```suggestion
					logrus.Fatalf("failed to read meta-file %s: %v", metaFilePath, err)
```